sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SyntaxHelper.cs

```diff
@@ -18,16 +18,28 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.Collections.Generic;
+using System.Linq;
 using Microsoft.CodeAnalysis;
 
 namespace SonarAnalyzer.Helpers
 {
     internal static class SyntaxHelper
     {
-        public static int GetLineNumber(this SyntaxToken token, bool isZeroBasedCount = true)
+        public static int GetStartLineNumber(this SyntaxToken token, bool isZeroBasedCount = true)
+        {
+            return GetLineNumbers(token, isZeroBasedCount).First();
```

Comment:
This is an expensive way of getting the first line number - would be better to just use the property and add the offset.

Reply:
This would be a bit slower, but not too much - the enumerable is lazily evaluated.

This method will be gone soon anyway.